unstructured/chunking/title.py

```diff
@@ -84,7 +84,7 @@ def chunk_by_title(
                                 item["start"] += start_char
                                 item["end"] += start_char
 
-                        if i > 0:
+                        if i > 0 and _value != value:
```

Comment:
can you remove this change? it's not present on main so I don't want to introduce it

Reply:
https://github.com/Unstructured-IO/unstructured/pull/1470#discussion_r1333392891

Reply:
ah just saw it thanks